app/uk/gov/hmrc/nationalinsurancerecord/connectors/CitizenDetailsConnector.scala

```diff
@@ -17,29 +17,24 @@
 package uk.gov.hmrc.nationalinsurancerecord.connectors
 
 import com.google.inject.Inject
-import play.api.Mode.Mode
-import play.api.http.Status._
-import play.api.{Configuration, Environment}
+import play.api.mvc.ControllerComponents
 import uk.gov.hmrc.domain.Nino
-import uk.gov.hmrc.http.{HeaderCarrier, HttpGet, HttpResponse, Upstream4xxResponse}
-import uk.gov.hmrc.nationalinsurancerecord.config.WSHttp
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse, Upstream4xxResponse}
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.nationalinsurancerecord.config.ApplicationConfig
 import uk.gov.hmrc.nationalinsurancerecord.services.MetricsService
-import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
-class CitizenDetailsConnector @Inject()(environment: Environment,
-                                        configuration: Configuration,
-                                        metrics: MetricsService) extends ServicesConfig{
-
-  val serviceUrl = baseUrl("citizen-details")
-  val http: HttpGet = new HttpGet with WSHttp
-
-  override protected def mode: Mode = environment.mode
-  override protected def runModeConfiguration: Configuration = configuration
+class CitizenDetailsConnector @Inject()(appConfig: ApplicationConfig,
+                                        http: HttpClient,
+                                        metrics: MetricsService,
+                                        cc: ControllerComponents) extends BackendController(cc) {
```

Comment:
BackendController is for controllers only

Reply:
Good spot, removed as isn't needed